doc/release/1.5.0-notes.rst

```diff
@@ -24,12 +24,35 @@ For running on PyPy, PyPy3 6.0+ and NumPy 1.15.0 are required.
 Highlights of this release
 --------------------------
 
--
+- wrappers for more than a dozen new ``LAPACK`` routines are now available
+  in `scipy.linalg.lapack`
```

Comment:
Do we want to acknowledge NumFOCUS' support of this effort?  (e.g. "thanks to the support of a NumFOCUS small development grant")

Reply:
I'm not aware of a precedent for this in our release notes for other things like GSOC, etc., so not sure I'm comfortable adding at this stage.